assets/js/blocks/price-filter/block.js

```diff
@@ -95,33 +110,30 @@ const PriceFilterBlock = ( { attributes, isEditor = false } ) => {
 	// Updates the query based on slider values.
 	const onSubmit = useCallback(
 		( newMinPrice, newMaxPrice ) => {
+			// Using snake_case, just for ease of assignment to URL params.
+			const finalMaxPrice =
+				newMaxPrice >= Number( maxConstraint )
+					? undefined
+					: newMaxPrice;
+			const finalMinPrice =
+				newMinPrice <= Number( minConstraint )
+					? undefined
+					: newMinPrice;
```

Comment:
This is extremely minor and just a suggestion, so feel free to ignore.

I personally find it slightly harder to read when the truthy assignment of a ternary operator is `undefined`. It _feels_ a bit more right that `undefined` would sorta be the fallback, you know what I mean? So, I'd more easily read:

```suggestion
			const finalMaxPrice =
				newMaxPrice < Number( maxConstraint )
					? newMaxPrice
					: undefined;
			const finalMinPrice =
				newMinPrice > Number( minConstraint )
					? newMinPrice
					: undefined;
```

Again, totally personal preference. Just throwing it out there.

Reply:
I can understand that. I'll reverse the logic here, I think the cognitive load is less like you suggested.